styles.css

```diff
@@ -0,0 +1,116 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  height: 100px;
+  left: 0;
+  top: 0;
+  background: #fff;
+}
```

Comment:
- When styling your webpage, it is important to start with the universal selector and give it a margin and padding of 0px, with box-sizing of border-box so as to prevent lots of unexpected margin/padding on the page.

You can do something like this 👇🏽 
```
* {
margin: 0,
padding: 0,
box-sizing: border-box
}
```

- Can we also remove all fixed width on your CSS?? You can work on your portfolio's spacing using the box model (padding, margin e.t.c)

Here is what your portfolio's design looks like 👇🏽 

| Your Design  |After clicking  Icon on Figma Design |
| :---:   | :---: | 
| <img width="418" alt="Screenshot 2022-12-01 at 18 33 47" src="https://user-images.githubusercontent.com/99246359/205125683-43d39b0e-8a95-45ba-88d5-bd5fe588db16.png"> |<img width="445" alt="Screenshot 2022-12-01 at 18 58 00" src="https://user-images.githubusercontent.com/99246359/205127110-53158258-bc3d-4a1a-9e9d-54a9697c9c27.png">  |

- Kindly fix the spacing using padding and margins 🌸

Reply:
Per your suggestion, I got rid of the fixed width and used margins and padding. It looks much better now. Nonetheless, I can't start the 'Let's connect' from the very beginning. Can you look at it?